src/courtroom.cpp

```diff
@@ -897,9 +897,12 @@ void Courtroom::enter_courtroom(int p_cid)
   }
 
   if (ao_app->custom_objection_enabled &&
-      (file_exists(ao_app->get_character_path(current_char, "custom1.gif")) ||
+      ((file_exists(ao_app->get_character_path(current_char, "custom1.gif")) ||
       file_exists(ao_app->get_character_path(current_char, "custom1.apng"))) &&
-      file_exists(ao_app->get_character_path(current_char, "custom1.wav")))
+      file_exists(ao_app->get_character_path(current_char, "custom1.wav"))) ||
+      ((file_exists(ao_app->get_character_path(current_char, "custom.gif")) ||
+      file_exists(ao_app->get_character_path(current_char, "custom.apng"))) &&
+      file_exists(ao_app->get_character_path(current_char, "custom.wav"))))
```

Comment:
This logic is overloaded.

Reply:
I know, this why I tried to not have the custom and custom1 as options for this reason but you told me to add backwards comp. I will probs end up scrapping this method anyway.